clients/ui/frontend/package.json

```diff
@@ -140,5 +140,14 @@
     "npm-run-all": "^4.1.5",
     "serve": "^14.2.4",
     "ts-jest": "^29.4.0"
-  }
+  },
+
```

Comment:
The dependencies are already listed [here](https://github.com/kubeflow/model-registry/blob/main/clients/ui/frontend/package.json#L93-L98). Can we delete these and bump the ones above?

You will also need to run `npm install` in `clients/ui/frontend` so we can also commit the updated package-lock.json.